.markdownlint.json

```diff
@@ -0,0 +1,15 @@
+{
+  "MD013": {
+    "code_blocks": false,
+    "headings": false,
+    "line_length": 80,
+    "tables": false
+  },
+  "MD024": {
+    "siblings_only": true
+  },
+  "MD031": true,
+  "MD034": true,
+  "MD040": true,
```

Comment:
Why bother including these if they are already on by default? Update this file to only include things that override markdownlint defaults.